src/components/pages/ProfilePage.tsx

```diff
@@ -31,7 +31,10 @@ export const ProfilePage = async () => {
   const { userId, name } = await getUserInfo();
 
   return (
-    <MobileLayout heading="Profile" headerRight={<ProfileHeaderDropdownMenuTriggerButton />}>
+    <MobileLayout
+      heading="Profile"
+      headerRight={<ProfileHeaderDropdownMenuTriggerButton userId={userId} />}
```

Comment:
Props drilling of `userId` starts from here, maybe better to refactor it in the future by using global state for example.